src/effects/VST/VSTEffect.h

```diff
@@ -199,6 +208,135 @@ struct VSTEffectWrapper : public VSTEffectLink, public XMLTagHandler
    void SaveFXP(const wxFileName& fn) const;
    void SaveFXProgram(wxMemoryBuffer& buf, int index) const;
 
+   // VST plugin -> host callback
+   static intptr_t AudioMaster(AEffect *effect,
+                               int32_t opcode,
+                               int32_t index,
+                               intptr_t value,
+                               void * ptr,
+                               float opt);
+
+   // These are needed to move the AudioMaster callback from VSTEffect to this struct
+   //
+   intptr_t mCurrentEffectID {};
+
+   bool mWantsIdle{ false };
+   std::unique_ptr<VSTEffectTimer> mTimer;
```

Comment:
It is not necessary to move these two into the Wrapper and I think they will eventually be in the Instance.

Reply:
Correct. Now moved back in the effect.